Shared/Samples/Animate 3D graphic/Animate3DGraphicView.Model.swift

```diff
@@ -137,7 +137,7 @@ extension Animate3DGraphicView {
         }
         
         deinit {
```

Comment:
See discussion in this thread: https://forums.swift.org/t/cannot-access-property-with-a-non-sendable-type-from-non-isolated-deinit/57877

Since the 2 `deinit` related changes are just cleaning up, I decided to loosely handle them with a separate `nonisolated` method that actually runs on MainActor or the inherited context.

Reply:
A better solution would be to move starting and stopping the animation from init and deinit to methods that the view could call on appear and disappear. That more closely ties lifetime of the animation to that of the view itself and eliminates the need for a task to stop it. Same elsewhere.